### PR TITLE
Fix issue #20684

### PR DESCRIPTION
**Pull request by doobrie**
Fixes issue #20684 

Removed the additional quote character that is displayed within the
Batch Admin UI.

### CODE REVIEW

**Comment by doobrie** [comment]
I've signed the Oracle Contributor Agreement to contribute to GassFish :)

**Comment by arindam-bandyopadhyay** [comment]
@glassfishrobot Run CI tests please

File: main/appserver/admingui/full/src/main/resources/batch/batchStepListing.jsf [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
     DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
 
-    Copyright (c) 2013 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2017 Oracle and/or its affiliates. All rights reserved.
```
Comment (sumauppa) [inline main/appserver/admingui/full/src/main/resources/batch/batchStepListing.jsf]: I think copyright should be 2013-2017, instead of 2017.

**Comment by arindam-bandyopadhyay** [comment]
@doobrie Please fix the copyright

**Review by sumauppa — COMMENTED** [review]
Please verify the copyright. Change looks good.

**Comment by doobrie** [comment]
Fixed the copyright date to read 2013-2017 as requested by @arindam-bandyopadhyay

**Comment by arindam-bandyopadhyay** [comment]
@glassfishrobot Run CI tests please

**Review by arindam-bandyopadhyay — APPROVED** [review]
Looks good

**Pull request merged**